presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java

```diff
@@ -1933,6 +1933,12 @@ private PhysicalOperation planGroupByAggregation(AggregationNode node, PhysicalO
                     .map(entry -> source.getTypes().get(entry))
                     .collect(toImmutableList());
 
+            if (memoryLimitBeforeSpill.toBytes() > 0) {
```

Comment:
In runtime check queries would nondeterministicaly fail with unsupported exception, and they would fail almost always with current version (with small memory limits before spill, prior to revocable memory). 

Since it's currently an experimental feature, that we expect to be turned off by default, I would prefer to fail fast and in deterministic fashion. At least until revocable memory is added.